demo/src/app/components/dropdown/demos/basic/dropdown-basic.html

```diff
@@ -0,0 +1,8 @@
+ <div ngbDropdown [open]="true">
+    <ngbd-button class="btn btn-primary" id="dropdownMenu1" ngbDropdownToggle>Toggle dropdown</ngbd-button>
```

Comment:
Why is this `ngbd-button`? Should it be a regular `<button>` element? 

Other thing: how about making it `btn-primary-outline` instead of `btn-primary`?